README.md

```diff
@@ -114,6 +197,12 @@ use the ``-v`` flag to drop old volumes.
 docker-compose rm -v
 ```
 
+Add the '-f' flag when using the mock compiler service:
+
+```bash
+docker-compose -f docker-compose-mock-compiler.yml rm -v
```

Comment:
Is that meant to remove the mock compiler service?

Reply:
It is my belief that many docker-compose commands rely on the config file when deciding what to do.

When I originally ran the basic cleanup command it looked like mock compiler was not getting cleaned up properly. It does not show up in 'docker-compose ps' output but it appears to linger in the 'docker ps -a' listing with an exit status.

lib-dlf2-maclap:arxiv-submission-ui dlf2$ docker ps -a
CONTAINER ID        IMAGE                 COMMAND                  CREATED             STATUS                            PORTS               NAMES
4793feb67df9        arxiv/mock-compiler   "pipenv run uwsgi --…"   41 hours ago        Exited (137) About a minute ago                       mock-compiler-api

When I add the -f argument this lingering entry gets removed.

lib-dlf2-maclap:arxiv-submission-ui dlf2$ docker ps -a
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES

I believe the config file might direct the cleanup process, and thus the mock compiler doesn't seem to get properlly cleaned up when using the non-mock default config.

I'm not sure if this matters all that much so I can remove this if you like. I just don't like the entry hanging around when the cleanup command cleaned out the rest. 

There is clearly a difference between 'docker-compose ps -a' output and 'docker ps -a'.

Reply:
Sounds good, might be good to update the README.md with a little explanation, e.g.

Add the '-f' flag when using the mock compiler service  to remove any lingering mock compiler image:

Reply:
Done